docker/manifest/gitea-1-0-0.yml

```diff
@@ -0,0 +1,14 @@
+image: sapk/gitea:1.0.0
```

Comment:
The only part missing is generation of image on arch. This could be done via qemu like https://github.com/multiarch/alpine does.